src/index.ts

```diff
@@ -254,18 +254,18 @@ yargs(Deno.args)
         );
 
         const { getGenerateFunction } = await import("./runners/runner.ts");
-        let generateFunction = await getGenerateFunction(
+        const limit = plimit(argv.llmConcurrency > 0 ? argv.llmConcurrency : 1);
```

Comment:
This now means there is no way to remove the limit

Reply:
@stwiname Fixed, I misunderstand = =.